db/migrate/20220430104830_create_comments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments do |t|
+
+      t.timestamps
+    end
```

Comment:
- [ ]  There is no migration column added. Make sure you have proper migration created for comments.
![image](https://user-images.githubusercontent.com/58504655/166104291-f1cc3e08-c381-43d7-995f-87b43b433a99.png)

Reply:
same here

Reply:
Same